### PR TITLE
Fix checkboxes with long labels overlapping nub

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: osu.Game/Graphics/UserInterface/OsuCheckbox.cs [hunk 1]
```diff
@@ -33,7 +32,6 @@ public Bindable<bool> Bindable
 
         public string LabelText
         {
-            get => labelSpriteText?.Text;
```
Comment (swoolcock) [inline osu.Game/Graphics/UserInterface/OsuCheckbox.cs]: Any reason for removing the getter?

Comment (peppy) [reply osu.Game/Graphics/UserInterface/OsuCheckbox.cs]: unsupported by `FillFlow` (and unused anywhere).

**Review by swoolcock — DISMISSED** [review]
LGTM

File: osu.Game/Graphics/UserInterface/OsuCheckbox.cs [hunk 2]
```diff
@@ -62,24 +60,28 @@ public OsuCheckbox()
...
                 new HoverClickSounds()
             };
 
             Nub.Current.BindTo(Current);
 
-            Current.DisabledChanged += disabled =>
-            {
-                labelSpriteText.Alpha = Nub.Alpha = disabled ? 0.3f : 1;
-            };
+            Current.DisabledChanged += disabled => { labelText.Alpha = Nub.Alpha = disabled ? 0.3f : 1; };
```
Comment (miterosan) [inline osu.Game/Graphics/UserInterface/OsuCheckbox.cs]: Remove paratheses

```suggestion
            Current.DisabledChanged += disabled => labelText.Alpha = Nub.Alpha = disabled ? 0.3f : 1;
```

**Review by miterosan — CHANGES_REQUESTED** [review]

**Review by smoogipoo — DISMISSED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**